src/coord/src/error.rs

```diff
@@ -119,7 +123,13 @@ impl fmt::Display for CoordError {
             CoordError::ReadOnlyParameter(p) => {
                 write!(f, "parameter {} cannot be changed", p.name().quoted())
             }
+            CoordError::ReadTimestampsDiffered => {
+                f.write_str("differing read timestamps in transaction")
+            }
             CoordError::SqlCatalog(e) => e.fmt(f),
+            CoordError::TailOnlyTransaction => {
+                f.write_str("TAIL in transactions must be only read statement")
```

Comment:
Can we word this message a bit better? I assume you mean "the only read statement"